src/ui/components/tusktask/group/TaskCard.tsx

```
@@ -0,0 +1,165 @@
+                    <div className="px-4 py-2 grid grid-cols-3 gap-2">
+                      <StatusOverviewCard
+                        title="Sub-Tasks"
+                        number={subTask.subTasks && subTask.subTasks.length}
+                        icon={ClockAlert}
+                      />
+                      <StatusOverviewCard
+                        title="Completed"
+                        number={
+                          subTask.subTasks &&
+                          subTask.subTasks.filter((task) => task.completedAt)
+                            .length
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)